umh-core/cmd/main.go

```
@@ -119,3 +136,33 @@
+		return
+	}
+
+	if config.Agent.CommunicatorConfig.APIURL != "" && config.Agent.CommunicatorConfig.AuthToken != "" {
+		// This can temporarely deactivated, e.g., during integration tests where just the mgmtcompanion-config is changed directly
+
+		login := v2.NewLogin(config.Agent.CommunicatorConfig.AuthToken, false)
+		if login == nil {
+			sentry.ReportIssuef(sentry.IssueTypeError, logger, "Failed to create login object")
+		}
+		communicationState.LoginResponse = login
+		logger.Info("Backend connection enabled, login response: ", zap.Any("login_name", login.Name))
```

golangci-lint (1.64.8)

SA5011: possible nil pointer dereference

(staticcheck)